filament/backend/src/vulkan/VulkanSwapChain.h

```diff
@@ -96,6 +96,7 @@ struct VulkanSwapChain : public HwSwapChain, fvkmemory::Resource {
     utils::FixedCapacityVector<fvkmemory::resource_ptr<VulkanTexture>> mColors;
     fvkmemory::resource_ptr<VulkanTexture> mDepth;
     VkExtent2D mExtent;
+    uint32_t mLayerCount;
```

Comment:
pls add initialization to 1 in the constructor, and if possible, also initialize  `mCurrentSwapIndex` to 0 as a side clean-up (thanks!)